cmd/antrea-agent/config.go

```diff
@@ -127,4 +127,10 @@ type AgentConfig struct {
 	// Provide the address of Kubernetes apiserver, to override any value provided in kubeconfig or InClusterConfig.
 	// Defaults to "". It must be a host string, a host:port pair, or a URL to the base of the apiserver.
 	KubeAPIServerOverride string `yaml:"kubeAPIServerOverride,omitempty"`
+	// Cipher suites to use on Client side.
```

Comment:
Do you use the upper case for "Client" on purpose?

Reply:
Yes.